soepy/python/pre_processing/model_processing.py

```diff
@@ -111,8 +132,15 @@ def group_parameters(init_dict, init_dict_flat):
         init_dict["PARAMETERS"]["delta_s3"],
     )
 
-    init_dict_flat["theta_p"] = init_dict["PARAMETERS"]["theta_p"]
-    init_dict_flat["theta_f"] = init_dict["PARAMETERS"]["theta_f"]
+    init_dict_flat["theta_p"] = [
```

Comment:
lines 135 - 140 can me merged with a for loop iterating over f and p, right?

Reply:
Yes. Done